plop-templates/pattern-library-page.hbs

```diff
@@ -0,0 +1,64 @@
+import { {{name}} } from '../../../../next';
+import Library from '../../Library';
+import Next from '../../LibraryNext';
+
+export default function {{name}}Page() {
+  return (
+    <Library.Page title="{{name}}" intro={<p>TODO</p>}>
+      <Library.Section
+        title="{{name}}"
+        intro={<p><code>{{name}}</code> is a {{category}} component</p>}
+      >
+        <Library.Pattern title="Status">
+          <p>
+            <code>{{name}}</code> is a presentational component...
+          </p>
+
+          <Library.Example title="Migrating to this component">
```

Comment:
This section is only needed if the component replaces some legacy component, right? Might be worth noting that in a Handlebars comment.